style.css

```diff
@@ -0,0 +1,995 @@
+  border: 2px solid #e51937;
+  color: #e51937;
+}
+
+.button__item_link,
+.button__item_link-info { text-transform: uppercase; }
+
+/* Второй экран */
+.section-desc,
+.text__wrap_character,
+.text__item,
+.text__item_small-wifi { display: flex; }
+
+.section-desc,
+.button { margin: auto; }
+
+.section-desc {
+  width: 95%;
+  height: 900px;
+}
+
+.section-up {
+  background: url(./Project/item_interior.svg) no-repeat;
+  background-position: 1% 100%;
+}
+
+.section-desc__wrap_up {
+  padding-top: 65px;
+  color: #e51937;
+}
+.section-desc__wrap { flex: .695; }
+
+.section-desc__title {
+  padding-left: 82px;
+  letter-spacing: 1px;
+}
+
+.text,
+.button__item_link-info,
+.button__item_link-white { color: #fff; }
+
+.text {
+  background-color: #e51937;
+  flex: .305;
+}
+
+.text__wrap_main {
+  margin-bottom: 65px;
+  padding-left: 51px;
+}
+
+.text__wrap_title {
+  margin-bottom: 11px;
+  padding-top: 54px;
+}
+
+.text__content {
+  width: 87%;
+  letter-spacing: .6px;
+}
+
+.text__wrap_character {
+  flex-direction: column;
+  justify-content: space-between;
+  height: 260px;
+  margin-bottom: 170px;
+  padding-left: 46px;
+}
+
+.text__item_small-screen,
+.text__item_small-bag { padding: 5px 0 0 5px; }
+.text__item_small-bag { margin-right: 24px; }
+
+.text__item_small-screen,
+.text__item_small-wifi { margin-right: 20px; }
+
+.text__item_bag,
+.text__item_small-wifi { padding-bottom: 4px; }
+
+.button { justify-content: center; }
+
+.button__item_big-right {
+  width: 36%;
+  padding-left: 34px;
+}
+
+.button__item_icon { margin-right: 10px; }
+
+.button__item_link-white { border: 2px solid #fff; }
+
+/* Третий экран */
+.section-down {
+  position: relative;
+  flex-direction: row-reverse;
+}
+
+.text__wrap_character-down,
+.text__wrap_down { padding-left: 58px; }
+
+.text__wrap_down {
+  margin-bottom: 55px;
+}
+
+.text__wrap_character-down {
+  height: 236px;
+  margin-bottom: 109px;
+}
+
+.text__wrap_section,
+.star,
+.star__wrapper,
+.score { display: flex; }
+
+.text__wrap_section,
+.star { justify-content: space-between; }
+
+.text__wrap_section {
+  flex-direction: column;
+  width: 96%;
+  height: 172px;
+}
+
+.star {
+  width: 80%;
+  text-transform: uppercase;
+}
+
+.star__item { margin-right: 3px; }
+
+.score {
+  margin-bottom: 41px;
+  padding-left: 67px;
+}
+
+.score__item_text { padding-top: 5px; }
+
+.score__item_icon {
+  margin-right: 25px;
+  padding-bottom: 19px;
+}
+
+.button_down {
+  justify-content: center;
+  width: 96%;
+  margin: 0;
+}
+
+.button__item_big-down {
+  width: 37%;
+  padding-left: 36px;
+}
+
+.section-desc__wrap-down {
+  background: url(./Project/modelS.svg) no-repeat;
+  background-position: 99% 19%;
+}
+
+.safe-body__item_corps {
+  position: relative;
+  right: 93px;
+  top: 172px;
+  z-index: 1;
+}
+
+.item__icon_corps { max-width: 110%; }
+
+/* Адаптив первой секции */
+
+/* Desktop */
+@media (max-width: 1245px) {
+  .header__wrapper_item,
+  .header__wrapper { flex: none; }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)